VMEncryption/main/oscrypto/rhel_72/encryptstates/PrereqState.py

```diff
@@ -19,8 +19,8 @@
 # Requires Python 2.7+
 #
 
-from OSEncryptionState import *
-from pprint import pprint
+from OSEncryptionState import OSEncryptionState
+import re
```

Comment:
Not sure this new import is needed (don't see it introduced in the diff below)

Reply:
same as above.